adapters/unicredit-adapter/src/main/java/de/adorsys/xs2a/adapter/service/impl/mapper/ScaStatusMapper.java

```diff
@@ -0,0 +1,59 @@
+package de.adorsys.xs2a.adapter.service.impl.mapper;
+
+import de.adorsys.xs2a.adapter.service.ScaStatus;
+import de.adorsys.xs2a.adapter.service.TransactionStatus;
+import de.adorsys.xs2a.adapter.service.ais.ConsentStatus;
+
+import java.util.HashMap;
+import java.util.Map;
+
+// Workaround for Berlin Group 1.1, as there is no Authorisation (SCA) as a separate model.
+// That's why we get consent status and have to map it to SCA status.
+class ScaStatusMapper {
+    private static final Map<ConsentStatus, ScaStatus> CONSENT_STATUS_TO_SCA_STATUS = new HashMap<>();
```

Comment:
Perfect opportunity to use java.util.EnumMap

Reply:
Agree, moved to `EnumMap`, thanks!